cypress/e2e/rooms_administration/memberManagementViaAdminPageWithDoubleRoleAdmin.feature

```diff
@@ -0,0 +1,108 @@
+@regression_test
+@stable_test
+@prio_0_dev
```

Comment:
why both prio tag here?

if feature is not available then use only `@prio_0_dev` not then use other one but not both of them